bookshelf/1-single-page/pom.xml

```diff
@@ -19,13 +19,13 @@ Copyright 2019 Google LLC
   <modelVersion>4.0.0</modelVersion>
   <packaging>war</packaging>
 
-  <groupId>com.example.flex.gettingstarted</groupId>
+  <groupId>com.google.cloud.samples</groupId>
```

Comment:
A lot of these are using `com.example.getstarted` to import - is that what this should be as well?